src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,125 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final Double MULTIPLIER = 1.5;
+    private Object[] internalArray;
+    private int size;
+
+    public ArrayList() {
+        this.internalArray = new Object[10];
```

Comment:
Create a constant for 10
```suggestion
        this.internalArray = new Object[DEFAULT_CAPACITY];
```